quickcomm/views.py

```diff
@@ -1,22 +1,33 @@
-from django.shortcuts import render, redirect
+from django.shortcuts import render, redirect, get_object_or_404
 from django.http import HttpResponse, HttpResponseRedirect
 from django.contrib.auth.decorators import login_required
 from django.contrib.auth import authenticate, login as auth_login, logout as auth_logout
-from quickcomm.forms import CreateMarkdownForm, CreatePlainTextForm, CreateLoginForm
+from django.core.paginator import Paginator
+from quickcomm.forms import CreateMarkdownForm, CreatePlainTextForm, CreateLoginForm, EditProfileForm
 from quickcomm.models import Author, Post
 
+
 # Create your views here.
 
+def get_current_author(request):
```

Comment:
Instead of having this method check for auth and returning None, you can just add the `@login_required` decorator to the function down below. That will automatically redirect to the login page if it is not working correctly.

Reply:
This should work in theory, but as of right now, as what I have doesn't assume the user is logged in this breaks things. Will fix when that is cleared up.